tests/README.md

```diff
@@ -33,12 +33,12 @@
 ## Getting Started
 
 The dependencies for the integration tests are collected and organized in
-dependency files in the `./requirements` folder, and separated into
+dependency files in the `./requirements-*` folders, and separated into
```

Comment:
is it only me or this sentence does not end well? separated into what?

Reply:
Separated into the table that is below.

Yeah, it is a bit strange but I'd rather let it be as it was :)